plan/main

```diff
@@ -2,4 +2,7 @@
 #include <turnkey/rails>
 
 libsqlite3-dev
-python-bcrypt
+python-setuptools
```

Comment:
Instead of doing it this way this, we should follow the process as noted in https://github.com/turnkeylinux/tracker/issues/1388 (i.e. via apt in the conf script).